en/docs/deploy-with-wso2-choreo.md

```diff
@@ -0,0 +1,8 @@
+# Deploy with WSO2 Choreo
+
+You can manage your Ballerina projects by deploying them in [WSO2 Choreo](https://wso2.com/choreo/). Integrating the [Choreo VS Code extension](https://marketplace.visualstudio.com/items?itemName=WSO2.choreo) with the Ballerina VS Code extension enables this functionality. 
+
+For detailed information, see the [WSO2 Choreo documentation](https://wso2.com/choreo/docs/develop-components/develop-components-using-vs-code/).
+
+<img src="https://wso2.com/ballerina/vscode/docs/img/choreo-integration/choreo-integration.png" class="cInlineImage-half"/>
```

Comment:
Seems this image have projects with improper names as `Bal service 1` etc. Have an image with some meaningful real world names.

Reply:
Also the name `test mono bal js` should be changed to a proper name from a real world example.

Reply:
Updated the image.